src/Track.h

```diff
@@ -237,10 +237,30 @@ class AUDACITY_DLL_API Track /* not final */
    , public AttachedTrackObjects
    , public std::enable_shared_from_this<Track> // see SharedPointer()
 {
+public:
```

Comment:
Lines were permuted in this header file, making the diff bigger and a little more difficult to review.